data-service/src/main/java/com/nova/dataservice/DTO/AppoinmentDTO.java

```diff
@@ -0,0 +1,77 @@
+package com.nova.dataservice.DTO;
+
```

Comment:
please dont use empty space remove this line.

Reply:
removed